### PR TITLE
Add size,seeds,leeches correct name parsing and daily search to CPasBien

### PR DESCRIPTION
**Pull request by duramato**
Kudos to @medariox for the help with size

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [x] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/providers/cpasbien.py [hunk 1]
```diff
@@ -105,7 +113,7 @@ def search(self, search_params, age=0, ep_obj=None):
 
                             items[mode].append(item)
 
-                except Exception as e:
+                except Exception, e:
```
Comment (medariox) [inline sickbeard/providers/cpasbien.py:116]: Should revert this change. Mistake?

Comment (duramato) [inline sickbeard/providers/cpasbien.py:116]: Yup dont even know how it got changed i didnt touched it..

Comment (medariox) [inline sickbeard/providers/cpasbien.py:116]: @duramato 
https://github.com/SickRage/SickRage/pull/513
